dts/arm/st/f4/stm32f429Xi.dtsi

```diff
@@ -24,4 +25,21 @@
 			};
 		};
 	};
+
+	fmc: memory-controller@a0000000 {
```

Comment:
Nodes definitions should be added in "soc dtsi files" (stm32f405.dtsi and stm32f446.dtsi in current case).
Current file is a "package dtsi file" and it aims at providing the possible memory variants of a SoC. Evrything that is common across variants should be defined at the lower level.